core/modules/layout/layout.admin.inc

```diff
@@ -423,6 +427,16 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
 }
 
 /**
+ * Sets the config settings to show all templates.
+ */
+function layout_settings_form_show_all_submit(&$form, &$form_state) {
+  $config = config('layout.settings');
+  $setting = !empty($form_state['show_all_admin']) ? 0 : 1;
+  $config->set('show_all_admin', $setting);
+  $config->save();
+}
```

Comment:
This function doesn't seem to be called, nor is there a form element for "show_all_admin". Seems like it may have intended to be removed?